Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -198,6 +198,17 @@ class PlaceDetailViewController: UIViewController {
     fileprivate func setBackgroundImage(toPhotoAtURL photoURLString: String?) {
         if let imageURLString = photoURLString,
             let imageURL = URL(string: imageURLString) {
+
+            let imageRequest = URLRequest(url: imageURL)
+            let cachedImage = UIImageView.sharedImageDownloader().imageCache?.imageforRequest(imageRequest, withAdditionalIdentifier: nil)
+
+            // Perform a cross fade between the existing/new images
+            let crossFade = CABasicAnimation(keyPath: "contents")
+            crossFade.duration = 0.4
+            crossFade.fromValue = backgroundImage.image?.cgImage
+            crossFade.toValue = cachedImage
+            backgroundImage.layer.add(crossFade, forKey: "animateContents")
+
             self.backgroundImage.setImageWith(imageURL)
```

Comment:
if we've already got the cached image, do we still need to use `backgroundImage.setImageWith(imageURL)` will this not just download the image again (although I suppose we'll just get it from the cache). But perhaps `self.backgroundImage.image = cachedImage`?